### ISSUE TITLE
EmptyFunctionBlock false positive: overridden function with comment in body

### ISSUE THREAD

**Issue by technoir42** [arturbosch/detekt#1684]
detekt 1.0.0-RC15

This gets reported by EmptyFunctionBlock as violation:
```kotlin
override fun fetch() {
    // No-op
}
```

There is `ignoreOverriddenFunctions` but I don't want to ignore all overridden functions, just ones with a comment in their body.

**Comment by rock3r** [comment]
I have the same. An example that triggers it:

```kotlin
private interface AnimationEndListener : Animator.AnimatorListener {
    override fun onAnimationStart(animation: Animator) {
        // No-op
    }
    override fun onAnimationCancel(animation: Animator) {
        // No-op
    }
    override fun onAnimationRepeat(animation: Animator) {
        // No-op
    }
}
```

It looks like what the [docs](https://arturbosch.github.io/detekt/empty-blocks.html#emptyfunctionblock) say: "This rule will not report functions overriding others.", is false, or more likely, the rule is bugged.

@technoir42 I think that config option is for the opposite case, that is, `open` functions that are meant to be overridden by subclasses.

**Comment by cortinico** [comment]
Yup I was able to reproduce it.
Apparently the logic use to check if the body had a comment or not was buggy. Moreover as @rock3r pointed out, the documentation was really confusing. Here the fix #1690
